src/emqtt_bench.erl

```diff
@@ -691,8 +694,8 @@ connect(Parent, N, PubSub, Opts) ->
                               undefined when TopicPayloadRend == undefined ->
                                    erlang:send_after(RandomPubWaitMS, self(), publish);
                               undefined ->
-                                 maps:foreach(fun(TopicName,  #{name := TopicName}) ->
-                                                    erlang:send_after(RandomPubWaitMS, self(), {publish, TopicName})
```

Comment:
I think the use of the random wait was intentional: to stagger publishing over a wider period of time instead of a huge simultaneous burst.

Reply:
This is for topic payload `--topics-payload` option, we don't want publisher send pkts immediately(regardless of random wait) after connect.

maybe change it to 
`RandomPubWaitMS ++ DelayMs` ?

Reply:
yes, that sounds good